### PR TITLE
DOC: Add welcome message to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by thewtex**
Scraped from https://github.com/nipy/nibabel/blob/master/.github/CONTRIBUTING.md

@jhlegarreta I was following the links you sent, and this stood out. Kudos to the nibabel community.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Contributing to ITK
 ===================
 
+Welcome to the ITK! We're excited you're here and want to contribute.
```
Comment (jhlegarreta) [inline CONTRIBUTING.md]: I'd reword the first part either as

`Welcome to ITK !`

or

`Welcome to the Insight Toolkit ! `

Comment (jhlegarreta) [reply CONTRIBUTING.md]: OK. looks like 131c019 addresses this. Thanks @thewtex .

Comment (thewtex) [reply CONTRIBUTING.md]: @jhlegarreta thanks for the review and suggestion -- updated to `Welcome to the Insight Toolkit (ITK) !`
